app/services/WeeklyPicker.scala

```diff
@@ -10,83 +10,16 @@ object WeeklyPicker {
 
```

Comment:
Nitpicking, but this tidying also makes it possible to tidy up imports here e.g. `import com.gu.memsub.images.{ResponsiveImageGenerator, ResponsiveImageGroup}`